templates/chartmuseum/chartmuseum-dpl.yaml

```diff
@@ -7,7 +7,13 @@ metadata:
 {{ include "harbor.labels" . | indent 4 }}
     component: chartmuseum
 spec:
+  {{- if and .Values.persistence.enabled (eq .Values.persistence.imageChartStorage.type "filesystem") }}
+  replicas: 1
+  strategy:
```

Comment:
How about we expose the `strategy` as a configuration item in `values.yaml`?

Reply:
Since we can't have more than one replica up at a time, we have to destroy old replica before creating a new one, which is what `Recreate` strategy does. We could expose strategy details for HA (multi-replica) case in a separate change.

Reply:
Thanks. I think we can still use the `RollingUpdate` as the default value. 
And please add some comment for the new added properties in both `value.yaml` and `README` to explain what it 's used for

Reply:
With RollingUpdate this Deployment will never get updated unless PVC has ReadWriteMany mode:
- first new pod is created
- new pod waits for PVC to get released by old pod
- old pod is not being killed by Deployment, because new one is not up yet.